backend/src/routes/getOtp.ts

```diff
@@ -0,0 +1,98 @@
+import { otp, registration } from '@/db/schema';
+import { sendMessage } from '@/utils/sms';
+import { desc, eq } from 'drizzle-orm';
+import type { FastifyInstance } from 'fastify';
+import otpGenerator from 'otp-generator';
+import * as schema from '../db/schema';
+
+interface GetOtpRequestBody {
+  phone_number: string;
+}
+
+// OTP validity duration : 15 mins
+const OTP_VALIDITY_TIME = 900;
+
+// as + symbol is not taken into query params
+// - + will be added after the number is processed
+export default function getOtp(fastify: FastifyInstance) {
+  fastify.get<{ Querystring: GetOtpRequestBody }>(
```

Comment:
In my opinion this operation should be a `POST` since it's a write operation.

Reply:
That way the phone number can go into the JSON body and have the `+` symbol.

Reply:
ok cool will change it